packages/core/src/hooks/useRawCalls.ts

```diff
@@ -1,10 +1,12 @@
 import { useEffect, useMemo } from 'react'
-import { RawCallResult, useChainState } from '../providers'
+import { RawCallResult } from '../providers'
 import { RawCall } from '../providers'
 import { Falsy } from '../model/types'
+import { useChainState } from './useChainState'
+import { QueryParams } from '../constants/type/Options'
 
-export function useRawCalls(calls: (RawCall | Falsy)[]): RawCallResult[] {
-  const { dispatchCalls, value } = useChainState()
+export function useRawCalls(calls: (RawCall | Falsy)[], queryParams: QueryParams): RawCallResult[] {
```

Comment:
Default to empty object

...and in the rest of the places where it's used

Reply:
if `RawCall` already has chainId do we have to pass it queryParams additionally?